datajunction-server/datajunction_server/api/nodes.py

```diff
@@ -842,14 +847,26 @@ async def link_dimension(
         name,
         raise_if_not_exists=True,
     )
+    if application_cache:
+        application_cache.set(
+            key=get_cache_key_for_node(
+                name=node.name,  # type: ignore
+                version=node.current_version,  # type: ignore
+                cached_value_type=CachedValueTypes.DIMENSIONS,
+            ),
+            value=None,
+            timeout=0,
```

Comment:
Maybe expose this `0` as a constant somewhere in the header to make it easier to find and adjust later?

Reply:
Hmm, I don't think we would ever change this value though. This is effectively deleting the entry in the cache by setting the value to `None` and the timeout to 0 so that it expires immediately. I could have added a `.delete(...)` method to the cache interface but I chose to keep that as simple as possible with just the `get` and `set`. What I'm saying is that if it was a constant it would basically be `ZERO=0` which isn't that helpful.